wallet/src/test/java/com/android/identity_credential/wallet/logging/EventLoggerTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.android.identity_credential.wallet.logging
+
+import com.android.identity_credential.wallet.util.MockActivityStorage
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertTrue
+
+class EventLoggerTest {
+
+    private lateinit var mockStorage: MockActivityStorage
+    private lateinit var activityLogger: EventLogger // Create an instance
+
+    @Before
+    fun setUp() {
+        mockStorage = MockActivityStorage()
```

Comment:
No need to mock `StorageEngine`, just use `EphemeralStorageEngine` instead.

Reply:
Done.